variants/Cargo.lock

```diff
@@ -13,13 +13,15 @@ dependencies = [
 name = "amazon-ssm-agent"
 version = "0.1.0"
 dependencies = [
+ "bash",
  "glibc",
 ]
 
 [[package]]
 name = "aws-dev"
 version = "0.1.0"
 dependencies = [
+ "amazon-ssm-agent",
```

Comment:
nit: ideally, the commit that modified the `aws-dev` variant should include the changes to the `Cargo.lock` file that add the SSM agent.